File: app/src/main/java/com/example/oneabes/AttendanceFragment.java
package com.example.oneabes;

import android.os.Bundle;

import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.CalendarView;
import android.widget.Toast;

import com.example.oneabes.databinding.FragmentAttendanceBinding;

public class AttendanceFragment extends Fragment {

    private FragmentAttendanceBinding binding;
    Long date;
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        binding = FragmentAttendanceBinding.inflate(inflater, container, false);
        View view = binding.getRoot();


        date = binding.cv.getDate();

        binding.cv.setOnDateChangeListener(new CalendarView.OnDateChangeListener(){
            public void onSelectedDayChange(CalendarView view, int year, int month, int dayOfMonth) {

                Toast.makeText(getContext(), "Year=" + year + " Month=" + month + " Day=" + dayOfMonth, Toast.LENGTH_LONG).show();

            }
        });

        return view;
    }
}

File: app/src/main/java/com/example/oneabes/AttendanceFragment.kt
package com.example.oneabes

import android.content.Intent
import android.view.LayoutInflater
import android.view.ViewGroup
import android.os.Bundle
import android.view.View
import android.widget.CalendarView.OnDateChangeListener
import android.widget.CalendarView
import android.widget.Toast
import androidx.fragment.app.Fragment
import com.example.oneabes.databinding.FragmentAttendanceBinding

class AttendanceFragment : Fragment() {
    private var binding: FragmentAttendanceBinding? = null
    var date: Long? = null
    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        // Inflate the layout for this fragment
        binding = FragmentAttendanceBinding.inflate(inflater, container, false)
        val view: View = binding!!.root
        date = binding!!.cv.date
        binding!!.cv.setOnDateChangeListener { view, year, month, dayOfMonth ->
//            Toast.makeText(
//                context, "Year=$year Month=$month Day=$dayOfMonth", Toast.LENGTH_LONG
//            ).show()
            startActivity(Intent(activity,Rv_CalenderDate::class.java))
        }
        return view
    }
}